AdyenComponents/PayTo/PayToComponent.swift

```diff
@@ -101,6 +140,52 @@ public final class PayToComponent: PaymentComponent,
         return item
     }()
 
+    /// The  account holder firstname text input item.
+    internal lazy var firstNameInputItem: FormTextInputItem = {
+        let item = FormTextInputItem(style: configuration.style.textField)
+        // TODO: Add translation
+        item.title = localizedString(LocalizationKey(key: "Account holder first name"), configuration.localizationParameters)
+        item.placeholder = localizedString(LocalizationKey(key: "Account holder first name"), configuration.localizationParameters)
+        item.identifier = ViewIdentifierBuilder.build(
+            scopeInstance: self,
+            postfix: ViewIdentifier.firstNameInputItem
+        )
+        return item
+    }()
+
+    /// The  account holder lastname text input item.
+    internal lazy var lastNameInputItem: FormTextInputItem = {
+        let item = FormTextInputItem(style: configuration.style.textField)
+        // TODO: Add translation
+        item.title = localizedString(LocalizationKey(key: "Account holder last name"), configuration.localizationParameters)
+        item.placeholder = localizedString(LocalizationKey(key: "Account holder last name"), configuration.localizationParameters)
+        item.identifier = ViewIdentifierBuilder.build(
+            scopeInstance: self,
+            postfix: ViewIdentifier.lastNameInputItem
+        )
+        return item
+    }()
+
+    /// The identifier picker item.
+    internal lazy var identifierPickerItem: FormStringPickerItem = {
+        let selectableValues = AccountIdentifiers.allCases.map { accountIdentifier in
+            FormStringPickerElement(identifier: accountIdentifier.identifier, title: accountIdentifier.description)
+        }
+
+        let item = FormStringPickerItem(
+            preselectedIdentifier: selectableValues[0],
```

Comment:
Could be a crash

Reply:
If the list is are empty accessing the first element will crash - ideally use `selectableValues.first` and try to unwrap the optional

Reply:
or when you create the array check for isEmpty but directly checking the first would be the clearest way.

Reply:
I have added an assertion :)